src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTab }) => {
+  const tabClick = title => {
+    onTabSelected(title);
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={selectedTab === tab.title ? 'is-active' : ''}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => tabClick(tab.title)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(tab => tab.title === selectedTab)?.content}
```

Comment:
```suggestion
        {selectedTab.content}
```
looks like you can just use selectedTab

Reply:
`selectedTab` it`s "Tab 1"